### ISSUE TITLE
Ability to export results elsewhere

### ISSUE THREAD

**Issue by fallc0nn** [MuShare/Httper-iOS#45]
Greetings!

Amazing app! It works very nice in my iPad!

I'd like to recommend a button to export and/or send the results to another app. Right now I have to manually select all the output, copy and paste elsewhere (in my iPad it does not show "select all" button).

**Comment by fallc0nn** [comment]
Zipped a small video showing this behavior in my ipad pro.

[httper-results.MOV.zip](https://github.com/MuShare/Httper-iOS/files/3776137/httper-results.MOV.zip)

**Comment by lm2343635** [comment]
You mean you need a button to copy all the content of the result son string?

**Comment by fallc0nn** [comment]
![D7A84261-B447-4081-8515-88D67A4234D3](https://user-images.githubusercontent.com/5271831/67669637-14e91e80-f951-11e9-916f-9a4257e0a7a7.jpeg)

Yes. This is a screenshot of a similar app as a example (yours is much better :)

**Comment by fallc0nn** [comment]
Yes. This is a screenshot of a similar app as a example (yours is much better :)

![6000CF20-0845-4F9A-8978-CE1A15042162](https://user-images.githubusercontent.com/5271831/67669982-b5d7d980-f951-11e9-9d71-60cced813b7a.jpeg)

**Comment by lm2343635** [comment]
I got it.
